src/main/java/dk/bec/bookanything/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,50 @@
+package dk.bec.bookanything.exception;
+
+import org.springframework.boot.context.properties.bind.validation.ValidationErrors;
```

Comment:
Remove not needed imports (could be done with ctrl + alt + o shortcut in inteliJ)